app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    linc = command.split()
+
+    if len(linc) == 3 and linc[0] == "mv" and not linc[2].count("/") == 0:
+        pathway = linc[2].split("/")[:-1]
+        name_file = linc[2].split("/")[-1]
+        linc_file = "/".join(pathway)
```

Comment:
Use os.path.join to concatenate parts of the path. In that case, your app will be cross-platform and will use either / or \ depending on OS